Slack-Apps/homework/responses/1.4-basic-slack-app.md

```diff
@@ -70,14 +72,18 @@ const result = await client.chat.postMessage({
     });
 ```
 
- The function will have JSON objects containing the information for the message. Look over [this documentation](https://api.slack.com/methods/chat.postMessage) on the Slack API to find which information you need to specify. Make sure to place the function within the try-catch statement to put it into effect.
+This code waits for a teammate to join before sending a message. The function will have JSON objects containing the information for the message. Look over [this documentation](https://api.slack.com/methods/chat.postMessage) on the Slack API to find which information you need to specify. Make sure to place the function within the try-catch statement to put it into effect.
 
 > **Note:** make sure to print the method to the console for testing purposes
 
-The output string should look something like "Welcome to the workspace, !" You can get the user id as a property of the event parameter. When finished, reinstall the app to your workspace. 
+The output string should look something like "Welcome to the workspace, !" You can get the user id as a property of the event parameter (refer to the end of [this tutorial](https://slack.dev/bolt-js/tutorial/getting-started#:~:text=%20Getting%20started%20with%20Bolt%20for%20JavaScript%20,handled%2C%20it%E2%80%99s%20time%20to%20set%20up...%20More%20%E2%80%A9http) for some clues). When finished, reinstall the app to your workspace. 
 
 If you have any or need some help, feel free to reach out to your TAs!
 
-**After you have finished, make sure to commit any file changes you made. To do this, please create a `branch` off of the master called `week(x)` with x being the # of the week you are on. For that week you will need to create a PR for every Learning Lab step to review and make sure your work is correct. For example: After finishing step 1.2, make a new branch off of `week(x)` and call it `week(x)-1.2` (with 1.2 being the step you're on). Then create a `Pull Request` between `week(x)` and `week(x)-1.2`. After reviewing you're PR or having another user review it, merge you're work and continue.**
+## Submission
+
+To move on to the following issue, commit your code! 
```

Comment:
can you specify which files you need to commit? and in a line summarize what the code is that they are submitting?

Reply:
Yup! I made the changes.